unobot.py

```diff
@@ -462,14 +463,14 @@ def get_card(self):
     @staticmethod
     def create_deck():
         new_deck = []
-        for card in COLORED_CARD_NUMS:
+        for card in (COLORED_CARD_NUMS + COLORED_CARD_NUMS[1:]):
             for color in CARD_COLORS:
                 new_deck.append(color + card)
         for card in SPECIAL_CARDS:
             new_deck.append(card)
             new_deck.append(card)
 
-        new_deck *= 4
+        new_deck *= 2
```

Comment:
Theoretically the right thing to do because nearly twice as many cards are generated now. Verify count though.